notebooks/Advanced_Usage.ipynb

```diff
@@ -37,7 +37,11 @@
     "  * [Accessing callback parameters](#Accessing-callback-parameters)\n",
```

Comment:
<p>I would have liked a heatmap of the activations to have a bit of a visual element but this is OK.</p>

 _Reply via <a href='https://app.reviewnb.com/skorch-dev/skorch/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='100'/>

Reply:
The activations have shape (batch_size, 5), that doesn't really lend itself too well for a visualizations. Also, running the notebook currently doesn't require matplotlib, it's probably not worth for this to add this requirement.